.github/workflows/v8build_not_for_windows.yml

```diff
@@ -0,0 +1,39 @@
+name: Automated V8 Build For MacOSX and Linux
```

Comment:
I don't think we need another workflow file. We could instead avoid cancelling the whole workflow when one job fails (https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast)

Reply:
Ohhh that is great. Thanks for linking up the documentation